mycore-viewer/src/main/typescript/modules/pdf/components/MyCoRePDFViewerComponent.ts

```diff
@@ -30,6 +30,8 @@ namespace mycore.viewer.components {
 
             // see MV-53
             (<any>PDFJS).disableAutoFetch=true;
+            (<any> PDFJS).cMapUrl=this._settings.webApplicationBaseURL + "/modules/iview2/cmaps/";
+            (<any> PDFJS).cMapPacked = true;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['PDFJS' is not defined. (no-undef)](https://app.codacy.com/app/MyCoRe/mycore/pullRequest?prid=4255860)